app/Actions/CompileAssets.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace App\Actions;
+
+use App\Shell\Shell;
+use Illuminate\Support\Arr;
+use Illuminate\Support\Facades\File;
+
+class CompileAssets
+{
+    use LamboAction;
+
+    protected $shell;
+
+    public function __construct(Shell $shell)
+    {
+        $this->shell = $shell;
+    }
+
+    public function __invoke()
+    {
+        if (config('lambo.store.mix') || config('lambo.store.full')) {
+
+            $this->squashProgress();
```

Comment:
Rename this method to `addSilentDevScript`

Reply:
Done.